tests/devices/test_default_qutrit_mixed.py

```diff
@@ -12,18 +12,43 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Tests for default qutrit mixed."""
+
+from functools import reduce
 import pytest
 import numpy as np
 
+
 import pennylane as qml
-from pennylane.devices.default_qutrit_mixed import DefaultQutritMixed
+from pennylane import math
+from pennylane.devices import DefaultQutritMixed, ExecutionConfig
+
+np.random.seed(0)
 
 
 def test_name():
     """Tests the name of DefaultQutritMixed."""
     assert DefaultQutritMixed().name == "default.qutrit.mixed"
 
 
+def test_shots():
```

Comment:
Can group these all into one class that focuses on testing generic device properties

Reply:
Yes, that's a good idea, thank you!